spec/truffle/methods/Digest::Base.singleton_class.txt

```diff
@@ -0,0 +1 @@
+inherited
```

Comment:
This is a difference from MRI, but I don't think it would be realistically be a problem given it's a standard hook in Ruby. The `inherited` hook is used to mix in `Truffle::Digest::Plugin` to new `Digest::Base` subclasses. This allows for seamless loading of plugins, but does introduce two user visible differences from MRI:

1. The presence of `inherited` in `Digest::Base.singleton_class.public_instance_methods(false)`
2. The mixed in `Truffle::Digest::Plugin` module for _digest_ gem plugins

I think both of these deviations are quite small and unlikely to cause any real world trouble. If they are problematic, we'll need to find another way to load _digest_ plugins.

Reply:
Yeah, agree